packages/angular/test/base/src/app/app.routes.ts

```diff
@@ -1,82 +1,6 @@
-import { AlertComponent } from './alert/alert.component';
-import { AccordionComponent } from './accordion/accordion.component';
 
 export const routes: Routes = [
-  { path: '', component: HomePageComponent },
-  { path: 'version-test', loadChildren: () => import('./version-test').then(m => m.VersionTestModule) },
-  { path: 'accordions', component: AccordionComponent },
-  { path: 'alerts', component: AlertComponent },
-  { path: 'inputs', component: InputsComponent },
-  { path: 'textarea', loadChildren: () => import('./textarea/textarea.module').then(m => m.TextareaModule) },
-  { path: 'searchbar', loadChildren: () => import('./searchbar/searchbar.module').then(m => m.SearchbarModule) },
-  { path: 'form', component: FormComponent },
-  { path: 'modals', component: ModalComponent },
-  { path: 'modal-inline', loadChildren: () => import('./modal-inline').then(m => m.ModalInlineModule) },
-  { path: 'view-child', component: ViewChildComponent },
-  { path: 'keep-contents-mounted', loadChildren: () => import('./keep-contents-mounted').then(m => m.OverlayAutoMountModule) },
-  { path: 'overlays-inline', loadChildren: () => import('./overlays-inline').then(m => m.OverlaysInlineModule) },
-  { path: 'popover-inline', loadChildren: () => import('./popover-inline').then(m => m.PopoverInlineModule) },
-  { path: 'providers', component: ProvidersComponent },
-  { path: 'router-link', component: RouterLinkComponent },
-  { path: 'router-link-page', component: RouterLinkPageComponent },
-  { path: 'router-link-page2/:id', component: RouterLinkPage2Component },
-  { path: 'router-link-page3', component: RouterLinkPage3Component },
-  { path: 'standalone', loadComponent: () => import('./standalone/standalone.component').then(c => c.StandaloneComponent) },
-  { path: 'tabs', redirectTo: '/tabs/account', pathMatch: 'full' },
-  {
-    path: 'navigation',
-    children: [
-      { path: 'page1', component: NavigationPage1Component },
-      { path: 'page2', component: NavigationPage2Component },
-      { path: 'page3', component: NavigationPage3Component }
-    ]
-  },
-  {
-    path: 'tabs',
-    loadChildren: () => import('./tabs/tabs.module').then(m => m.TabsPageModule)
-  },
-  {
-    path: 'tabs-global',
-    loadChildren: () => import('./tabs-global/tabs-global.module').then(m => m.TabsGlobalModule)
-  },
-  {
-    path: 'tabs-slots',
-    loadComponent: () => import('./tabs-slots.component').then(c => c.TabsSlotsComponent)
-  },
-  {
-    path: 'nested-outlet',
-    component: NestedOutletComponent,
-    children: [
-      {
-        path: 'page',
-        component: NestedOutletPageComponent
-      },
-      {
-        path: 'page2',
-        component: NestedOutletPage2Component
-      }
-    ]
-  },
-  {
-    path: 'form-controls/range',
-    loadChildren: () => import('./form-controls/range/range.module').then(m => m.RangeModule)
-  }
+  { path: 'lazy', loadChildren: () => import('./app-lazy/app.module').then(m => m.AppModule) },
```

Comment:
~ Should we add a default match path to redirect to a known route? Otherwise when serving the app you need to know to go to `/lazy/something` or `/standalone/something`?

e.g.:
```
{
  path: '',
  pathMatch: 'full',
  redirectTo: '/lazy'
}
```

Reply:
Good call. I updated this in https://github.com/ionic-team/ionic-framework/pull/27867/commits/85cac110653bbb6170e5f93143f6f6338a368266